examples/solids/problems/finite-strain-neo-hookean-initial-1-ad.c

```diff
@@ -0,0 +1,60 @@
+#include <ceed.h>
+#include "../include/structs.h"
+#include "../include/setup-libceed.h"
+#include "../problems/problems.h"
+#include "../problems/neo-hookean.h"
+#include "../qfunctions/common.h"
+#include "../qfunctions/finite-strain-neo-hookean-initial-1-ad.h"
+
+static const char *const field_names[] = {"gradu", "Swork", "tape"};
+static CeedInt field_sizes[] = {9, 6, 1};
+
+ProblemData finite_strain_neo_Hookean_initial_1_ad = {
+  .setup_geo = SetupGeo,
+  .setup_geo_loc = SetupGeo_loc,
+  .geo_data_size = 10,
+  .quadrature_mode = CEED_GAUSS,
+  .residual = ElasFSInitialNH1F_AD,
+  .residual_loc = ElasFSInitialNH1F_AD_loc,
+  .number_fields_stored = sizeof(field_sizes) / sizeof(*field_sizes),
```

Comment:
Probably worth propagating this change to all problems.

Reply:
Will do!